### DISCUSSION TITLE
Partition Coefficient

### DISCUSSION THREAD

**Discussion by pburkat** [Open-Systems-Pharmacology/Forum#1031] [Questions & Problems] [answered]
Hello, is it possible to directly enter tissue:plasma partition coefficients from experimental values into a simulation rather than having them calculated by the methods used by PK-Sim?

**Comment by Yuri05** [comment, marked as answer]
Yes, you can change partition coefficients in a simulation. You can find them under `Distribution=>Partition coefficient`
<img width="500" src="https://user-images.githubusercontent.com/25061876/143850032-074c135f-95d5-435b-85e3-b38058631bdb.png">

**Reply by pburkat** [reply]
Thanks.
